server/server.cpp

```diff
@@ -31,9 +31,12 @@
 // Stuff to wait for input on stdin.
 #ifdef Q_OS_WIN
 #include <QMutexLocker>
+#include <io.h>
 #else
 #include <QtCore/QSocketNotifier>
+#include <unistd.h>
```

Comment:
Why do we need `unistd` on Linux and Mac?

Reply:
> Why do we need `unistd` on Linux and Mac?

https://github.com/longturn/freeciv21/blob/f779ae6b04739d613b5077c47aebc550f9657fc1/server/server.cpp#L276

Reply:
Thanks :) It's obviously already included transitively but better do it explicitly indeed.